src/cow-react/utils/amountFormat/index.test.ts

```diff
@@ -125,7 +125,7 @@ describe('Amounts formatting', () => {
       const result2 = formatFiatAmount(getAmount('60001444', 3))
 
       expect(result1).toBe('7,344,360.23')
-      expect(result2).toBe('60.00B')
+      expect(result2).toBe('60B')
```

Comment:
I believe in this case it should trim

Reply:
I'm not sure, I guess we did it on purpose (`'Precision for fiat amounts is always 2`)

Reply:
However, I agree, it doesn't make sense to display trailing zeros